Dockerfile

```diff
@@ -102,12 +102,16 @@ COPY cli ./
 
 # Let's now simply use 'pip' to install semgrep.
 # Note the difference between .run-deps and .build-deps below.
-# TODO? what does --virtual= mean? why all in one command below?
+# We use a single command to install packages, install semgrep, and remove
+# packages to keep a small Docker image (classic Docker trick).
+# Here is why we need the apk packages below:
+#  - build-base: ??
+#  - make, g++: to compile the jsonnet C++ library which is installed
+#    by 'pip install jsonnet'
 # TODO? why the mkdir -p /tmp/.cache?
-# TODO: for jsonnet, you need for the build-deps before 'pip install jsonnet'
-#  'apk add g++ make'
 # hadolint ignore=DL3013
-RUN apk add --no-cache --virtual=.build-deps build-base &&\
+RUN apk add --no-cache --virtual=.build-deps build-base make g++ &&\
+     pip install jsonnet &&\
```

Comment:
Should we add it as a dependency in setup.py or are you trying to keep this internal use only for testing for now?

Reply:
https://github.com/returntocorp/semgrep/blob/develop/cli/setup.py#L108

Reply:
yep internal, next step would be setup.py

Reply:
I can add a comment about it